index.html

```diff
@@ -9,78 +9,100 @@
 auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org –> 
 
 
+<!DOCTYPE html>
+<html lang = "en-US">
+ <head>
+ <meta charset = "UTF-8">
+ <title>index.html</title>
+ <link rel = "stylesheet"
+   type = "text/css"
+   href = "style.css"
+/>
 
-'''html
-<header>
-  <div>
+<!-- ||||||||||||||| Header ||||||||||||||| -->
+
+    <header>
     <div>
-      <img class='blackheart' src='https://github.com/ironhack-labs/lab-html-exercise/blob/06f6124f5e3232d861ff975de5348f81839217a6/images/black-heart.png?raw=true' alt='Black Heart' />
-      <span> Nifty Penguin Magic </span>
-    </div>
-        <nav>
-        <ul>
+        <div>
+        <img class='blackheart' src='https://github.com/ironhack-labs/lab-html-exercise/blob/06f6124f5e3232d861ff975de5348f81839217a6/images/black-heart.png?raw=true' alt='Black Heart' />
```

Comment:
You should use a relative path when you're looking to show images like `<img src="images/black-heart.png"`